### DISCUSSION TITLE
after change ip of server , user not work anymore

### DISCUSSION THREAD

**Discussion by fpgi** [angristan/openvpn-install#1402] [Help]
Hi
some datacenter have this ability to change ip of server like Hetzner 
after change ip and start server , my clients can not connect to server 
i change /etc/openvpn/server/server.conf ( i think it is exact file of openvpn configuration ) and change config file of user but still can not connect . 
i must uninstall openvpn via script and install that again and after that it work . which files must change for successful connection of user after change ip ?

**Comment by AlpixTM** [comment]
Did you recreate the client files? They include the IP of the server the client connects to, so they will still connect to the old IP

**Comment by fpgi** [comment]
yes i changed config file of customer to new ip 
i didn't create new user . just edit config file of customer
